src/builder.ts

```diff
@@ -157,6 +157,12 @@ export interface Options {
    */
   scalars?: Partial<Record<Typegen.GetGen<'scalars', string>, GraphQLScalarType>>
   computedInputs?: GlobalComputedInputs
+  /**
+   * Graphql does not support union types, atomic operation are enabled by default for CRUD update and upsert
+   *
+   * @default false
+   */
+  disableAtomicOperations?: boolean
```

Comment:
Remove a level of negation by calling this `atomOperations` and make its default `true`.